src/db/db.go

```diff
@@ -0,0 +1,106 @@
+package db
```

Comment:
Could you please refactor this file a bit for better readability - I would move the DB queries to a file named queries.go and the connection to the DB to a file called migration.go, that way we separate between the DB operation logic and the logic within this file to listen to the channels and update the database accordingly?